runtime/doc/eval.txt

```diff
@@ -5266,5 +5275,113 @@ Usage: >vim
 	:call dist#vim9#Launch(<args>)
 	:Launch <app> <args>.
 <
-
+==============================================================================
+15. Clipboard providers				*clipboard-providers*
+
+The clipboard provider feature allows the "+" and "*" registers to be
+overrided by custom Vimscript functions.  There can be multiple providers, and
+Vim chooses which one to use based on 'clipmethod'.  Despite the name, it does
+not use the 'clipboard' option and should be treated separate from the
+clipboard functionality.  It essentially overrides the existing behaviour of
+the clipboard registers.
+
+						*clipboard-providers-no-clipboard*
+If the |+clipboard| feature is not enabled, then the "+" and "*" registers
+will not be enabled/available unless |v:clipmethod| is set to a provider.
```

Comment:
can you add something like:

This allows Vim to access the system clipboard via an external provider
even if the |+clipboard| feature is not compiled in.